src/components/ProfileInfo/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const ProfileInfo = () => {
+	const { user } = useSelector((state) => state.userReducer);
+
+	return (
+		<>
+			<div className="container d-flex justify-content-center profile-box">
+				<div id="profie-pic">
+					{user.avatar_url ? <img src={user.avatar_url} alt="User's Profile Picture" /> : <p></p>}
+				</div>
+				<div className="profile-data">
+					<h1 className="username">{user.login}</h1>
+					<h3>{user.location ? user.location : ''}</h3>
+					{user.html_url ? (
+						<a className="visit-link" href={user.html_url} target="_blank">
+							Visit
+						</a>
+					) : (
```

Comment:
so if the user url is not available, you just throw an empty string?
its the same as the above as well, if their location is not available, you just throw out an empty string?

Reply:
Yea its a ternary statement, so if theres no location provided it returns empty and if there is it posts it